site/app/libraries/QueueItem.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace app\libraries;
+
+/**
+ * Class QueueItem
+ * Helper class used to interpret the files read in the autograding json files
+ */
+class QueueItem {
+    /** @prop-read @var array */
```

Comment:
`@prop-read` does not do anything here as this is not a model. This probably could just be a model rather than a library since it just stores data for you.